src/System.Private.CoreLib/src/System/Runtime/CompilerServices/ClassConstructorRunner.cs

```diff
@@ -94,7 +94,7 @@ public static unsafe void EnsureClassConstructorRun(StaticClassConstructionConte
                             {
                                 NoisyLog("Calling cctor, cctor={0}, thread={1}", pfnCctor, currentManagedThreadId);
 
-                                Call<int>(pfnCctor);
+                                Call(pfnCctor);
```

Comment:
I tested this change on ProjectN and it works. We probably just got away with it because the incorrect return value gets ignored.